scripts/XLSXHandler.py

```diff
@@ -187,7 +187,12 @@ def __handleConditionBlock(self, intent, block, domain):
                 if row[0] and conditionHasX:
                     intent = re.sub(Dialog.X_PLACEHOLDER, row[0], conditionTemplate)
                     intentData = self._dialogData.getIntentData(intent, domain)
-                intentData.addRawOutput(row[1:], self._labelsMap)
+                # check for empty output and print message with error location info
+                if isinstance(row[1:], tuple) and sum([1 if x else 0 for x in row[1:]]) > 0:
+                    intentData.addRawOutput(row[1:], self._labelsMap)
+                else:
+                    eprintf('Warning: Empty response at: ' + intent + '\n')
+
```

Comment:
better warning message that helps to locate the source of warning in excel file.
Needed to be done outside of addRawOutput where location data are not passed.

Reply:
Removed as source changed a lot.